hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -426,7 +426,7 @@ onOpenNetworkReqSn env ledger st otherParty sv sn requestedTxIds mDecommitTx =
                       { headId
                       , version = version
                       , number = sn
-                      , confirmed = requestedTxIds
+                      , confirmed = requestedTxs
```

Comment:
Why ids are not enough anymore?

Reply:
It simplifies the client because we can just process `SnapshotConfirmed` messages as if it were blocks on a block chain. Before, we had snapshot confirmed only contain ids and `TxValid` contain the actual transaction.

This is a short-coming in `hydraw` too. There we treat `TxValid` as as pixel painted, although it is not even confirmed!